src/widgets/vitals/vitals-overview.css

```diff
@@ -1,6 +1,7 @@
 .vitalsTable {
   width: 100%;
   font-family: "Work Sans";
+  overflow-x: scroll;
```

Comment:
Should this always be like that? Or do you want to have `auto` here? `scroll` will always display a scrollbar - even if it is "disabled", because there is nothing to scroll.